app/routes/bestuurseenheid/zitting/notulen.js

```diff
@@ -1,18 +1,26 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidZittingNotulenRoute extends Route {
-  breadCrumb = { title: 'Notulen' };
+  @service store;
 
   async model() {
-    const meeting = this.modelFor('bestuurseenheid.zitting');
-    const meetingNotes = await this.store.query('notulen', {
-      "filter[zitting][:id:]": meeting.id,
-      sort: "-publication.created-on",
-      include: "publication"
-    });
-
-    const notulen = meetingNotes.firstObject;
-    return { notulen, zitting: meeting };
+    //Why was it done this way:
+    //const meeting = await this.modelFor('bestuurseenheid.zitting');
+    //const meetingNotes = await this.store.query('notulen', {
+    //  'filter[zitting][:id:]': meeting.id,
+    //  sort: '-publication.created-on',
+    //  include: 'publication',
+    //});
+    //const notulen = meetingNotes.firstObject;
+    //return { notulen, zitting: meeting };
+    //When this looks easier and works the same:  ?
```

Comment:
This seems to do extra calls since it doesn't use includes?

Reply:
some history: in an earlier version it was possible to do multiple publications and the above makes sure you have the most recent one.